### ISSUE TITLE
Dostęp do zasobu /messaging/threads

### ISSUE THREAD

**Issue by Atospl** [allegro/allegro-api#4999]
Dzień dobry,

po wysłaniu zapytania GET do zasobu /messaging/threads otrzymuję zwrot:

'{"errors":[{"code":"AccessDenied","message":"Access is denied","details":null,"path":null,"userMessage":"Access denied. Contact the author of the application."}]}'

Czy to oznacza, że aby korzystać z zasobu muszę mieć podpisaną umowę NDA z Allegro? Z góry dziękuję za odpowiedź

**Comment by kfijalko** [comment]
Token musi mieć uprawnienie: allegro:api:messaging.

**Comment by Atospl** [comment]
Dzięki za sugestię, ale to nie to; token który w tej chwili wykorzystałem posiada to uprawnienie:
...
,"expires_in":43199,"scope":"allegro:api:orders:read allegro:api:profile:write allegro:api:sale:offers:write allegro:api:billing:read allegro:api:campaigns allegro:api:disputes allegro:api:sale:offers:read allegro:api:bids allegro:api:orders:write allegro:api:ads allegro:api:payments:write allegro:api:sale:settings:write allegro:api:profile:read allegro:api:ratings allegro:api:sale:settings:read allegro:api:payments:read **allegro:api:messaging**","allegro_api":true,...}

**Comment by MartaNowaczyk** [comment]
Nie musisz mieć podpisanej umowy NDA, wystarczy odpowiedni scope i odpowiednia autoryzacja. 
Może używasz autoryzacji typu [client credentials](https://developer.allegro.pl/auth/#client-credentials-flow), która nie pozwala na dostęp do tego zasobu?

**Comment by Atospl** [comment]
Nie, używam ścieżki Device Flow. Identyfikator zapytania: {'trace-id': 'cedca51e100ae7e4', 'content-type': 'application/json; charset=utf-8', 'content-length': '162', 'date': 'Mon, 04 Oct 2021 12:23:31 GMT'}

**Comment by MartaNowaczyk** [comment]
Po trace-id widzę w naszych logach brak scope'a: allegro:api:messaging

**Comment by Atospl** [comment]
Dziękuję, zakładałem że jeśli nie przekazuję scope'a przy tworzeniu tokenów, to otrzymuję wszystkie.
